uint32 session_id
uint32 camera_id
uint32 csaliency_id
time ts
Float32XY position
float32 confidence
